config.json

```diff
@@ -67,6 +67,7 @@
       }
     ],
     "show_channels": false,
+    "show_language_selector": true,
```

Comment:
We should default this to `false` to avoid unintentionally enabling the feature on existing merchant stores during upgrades without user consent.

Reply:
Of course, you are absolutely right, I am going to set this parameter to `false` after I remove the "DRAFT" label from the PR and do some refactoring.